app/Controllers/feedController.php

```diff
@@ -365,15 +367,18 @@ public static function actualizeFeed($feed_id, $feed_url, $force, $simplePiePush
 						// This entry should not be added considering configuration and date.
 						$oldGuids[] = $entry->guid();
 					} else {
+						$read_upon_reception = $feed->$attributes['read_upon_reception'] !== null
```

Comment:
@Frenzie Do you know how those lines are supposed to be written to pass Travis / PHPCS?
I have already tried two variations without success...

Reply:
Unfortunately it wants it all to be on one line like this:

```diff
diff --git a/app/Controllers/feedController.php b/app/Controllers/feedController.php
index eb82a797..aa322a8b 100755
--- a/app/Controllers/feedController.php
+++ b/app/Controllers/feedController.php
@@ -353,9 +353,7 @@ class FreshRSS_feed_Controller extends Minz_ActionController {
                                                } else {        //This entry already exists but has been updated
                                                        //Minz_Log::debug('Entry with GUID `' . $entry->guid() . '` updated in feed ' . $feed->id() .
                                                                //', old hash ' . $existingHash . ', new hash ' . $entry->hash());
-                                                       $mark_updated_article_unread = $feed->$attributes['mark_updated_article_unread'] !== null
-                                                               ? $feed->$attributes['mark_updated_article_unread']
-                                                               : FreshRSS_Context::$user_conf->mark_updated_article_unread;
+                                                       $mark_updated_article_unread = $feed->$attributes['mark_updated_article_unread'] !== null ? $feed->$attributes['mark_updated_article_unread'] : FreshRSS_Context::$user_conf->mark_updated_article_unread;
                                                        $needFeedCacheRefresh = $mark_updated_article_unread;
                                                        $entry->_isRead(FreshRSS_Context::$user_conf->mark_updated_article_unread ? false : null);      //Change is_read according to policy.
                                                        if (!$entryDAO->inTransaction()) {
@@ -367,9 +365,7 @@ class FreshRSS_feed_Controller extends Minz_ActionController {
                                                // This entry should not be added considering configuration and date.
                                                $oldGuids[] = $entry->guid();
                                        } else {
-                                               $read_upon_reception = $feed->$attributes['read_upon_reception'] !== null
-                                                       ? $feed->$attributes['read_upon_reception']
-                                                       : FreshRSS_Context::$user_conf->mark_when['reception'];
+                                               $read_upon_reception = $feed->$attributes['read_upon_reception'] !== null ? $feed->$attributes['read_upon_reception'] : FreshRSS_Context::$user_conf->mark_when['reception'];
                                                if ($isNewFeed) {
                                                        $id = min(time(), $entry_date) . uSecString();
                                                        $entry->_isRead($read_upon_reception);
```

So for legibility it might be better to use an if/else statement. I could also try to find out if there's a way to coax PHPCS into accepting this if you like but I don't know if that's an option.